yt/geometry/coordinates/spherical_coordinates.py

```diff
@@ -81,6 +81,53 @@ def _path_phi(field, data):
             units="code_length",
         )
 
+        self.setup_cartesian_fields(registry)
+
+    def setup_cartesian_fields(self, registry):
```

Comment:
this doesn't seem to be an existing method name in other CoordinateHandler classes and I can't see what we get from splitting this into a separate method, is there any benefit I'm not seeing ?

Reply:
I'm okay with it being in the same function. I was using this for debugging, and wasn't sure if we wanted to make it a property of the superclass.